sled-agent/src/config.rs

```diff
@@ -5,26 +5,30 @@
 //! Interfaces for working with sled agent configuration
 
 use crate::common::vlan::VlanID;
-use crate::illumos::dladm::PhysicalLink;
+use crate::illumos::dladm::{self, Dladm, PhysicalLink};
 use crate::illumos::zpool::ZpoolName;
-use dropshot::ConfigDropshot;
 use dropshot::ConfigLogging;
+use omicron_common::api::external::Ipv6Net;
 use serde::Deserialize;
-use std::net::SocketAddr;
+use std::net::{Ipv6Addr, SocketAddr, SocketAddrV6};
 use std::path::Path;
 use uuid::Uuid;
 
+pub const SLED_AGENT_PORT: u16 = 12345;
+
+/// Given a subnet, return the sled agent address.
+pub(crate) fn get_sled_address(subnet: Ipv6Net) -> SocketAddrV6 {
+    let sled_agent_ip = Ipv6Addr::from(u128::from(subnet.ip()) + 1);
```

Comment:
We might want to use [the iterator](https://docs.rs/ipnetwork/latest/ipnetwork/struct.Ipv6NetworkIterator.html) provided by `Ipv6Network` to do this, since that automatically uses the network address and the subnet mask appropriately. I.e., `subnet.iter().next().unwrap()`.

Reply:
Done